docs/tutorials/longer-metrics-storage.md

```diff
@@ -7,30 +7,27 @@ Many people think Netdata can only store about an hour's worth of real-time metr
 configuration today. With the right settings, Netdata is quite capable of efficiently storing hours or days worth of
 historical, per-second metrics without having to rely on a [backend](../../backends/).
 
-This tutorial gives two options for configuring Netdata to store more metrics. We recommend the [**database
-engine**](#using-the-database-engine), as it will soon be the default configuration. However, you can stick with the
-current default **round-robin database** if you prefer.
+This tutorial gives two options for configuring Netdata to store more metrics. **We recommend the default [database
+engine](#using-the-database-engine)**, but you can stick with or switch to the round-robin database if you prefer.
 
 Let's get started.
 
 ## Using the database engine
 
 The database engine uses RAM to store recent metrics while also using a "spill to disk" feature that takes advantage of
-available disk space for long-term metrics storage.This feature of the database engine allows you to store a much larger
-dataset than your system's available RAM.
+available disk space for long-term metrics storage. This feature of the database engine allows you to store a much
+larger dataset than your system's available RAM.
 
-The database engine will eventually become the default method of retaining metrics, but until then, you can switch to
-the database engine by changing a single option.
-
-Edit your `netdata.conf` file and change the `memory mode` setting to `dbengine`:
+The database engine is currently the default method of storing metrics, but if you're not sure which database you're
+using, check out your `netdata.conf` file and look for the `memory mode` setting:
 
 ```conf
 [global]
     memory mode = dbengine
 ```
 
-Next, restart Netdata. On Linux systems, we recommend running `sudo service netdata restart`. You're now using the
-database engine!
+If `memory mode` is set to anything but `dbengine`, change it and restart Netdata. On Linux systems, we recommend
+running `sudo service netdata restart`. You're now using the database engine!
```

Comment:
The command "service" is not available in all Linuxes, it only exists on distribution that has ```systemd``` , is it not better to add an information after the service instruction for these distributions, something like "case you have systemd on your Linux or use the common command to restart Netdata on your distribution."?